src/Document.tsx

```diff
@@ -11,6 +11,8 @@ import {
 } from '@chakra-ui/react'
 import { RouteComponentProps } from '@reach/router'
 import * as React from 'react'
+import { useState } from 'react'
+import { useEffect } from 'react'
```

Comment:
Could pull these into a single import. I guess there's even a way to put them all into the same import as the main React one?

Reply:
Good call, will include in next commit!